### PR TITLE
Fix storage display

### PR DESCRIPTION
**Pull request by icerove**
fix the bug for 'from genesis'

### CODE REVIEW

File: frontend/src/components/accounts/AccountDetails.tsx [hunk 1]
```diff
@@ -119,9 +120,13 @@ export default class extends React.Component<Props, State> {
             <CardCell
               title="Creation Hash"
               text={
-                <TransactionLink transactionHash={account.address}>
-                  {account.address}
-                </TransactionLink>
+                account.address === message ? (
```
Comment (frol) [inline frontend/src/components/accounts/AccountDetails.tsx]: We should just make that string empty wherever we create a "dummy" account object, and check if it is an empty string here and then display a human-readable message here.

File: frontend/src/components/accounts/AccountDetails.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ export default class extends React.Component<Props, State> {
   render() {
     const { account } = this.props;
     const { storagePaidAtBlockHash } = this.state;
+    const message = "from genesis" || "loading";
```
Comment (frol) [inline frontend/src/components/accounts/AccountDetails.tsx]: This does not make any sense to me... Well, given ^ we should just remove this. I still wonder what this expression was meant to be.

File: frontend/src/components/accounts/AccountDetails.tsx [hunk 3]
```diff
@@ -119,9 +119,13 @@ export default class extends React.Component<Props, State> {
             <CardCell
               title="Creation Hash"
               text={
-                <TransactionLink transactionHash={account.address}>
-                  {account.address}
-                </TransactionLink>
+                account.address === "" ? (
+                  ""
```
Comment (frol) [inline frontend/src/components/accounts/AccountDetails.tsx]: Should we display "from genesis" here?

Comment (icerove) [reply frontend/src/components/accounts/AccountDetails.tsx]: maybe not, let's wait to show real information about genesis when we finish fetching from genesis?

Comment (frol) [reply frontend/src/components/accounts/AccountDetails.tsx]: Ok. Fair enough.

**Pull request merged**